.github/CODEOWNERS

```diff
@@ -108,7 +108,7 @@
 
 # ServiceLabel: %Communication - Phone Numbers
 # PRLabel: %Communication - Phone Numbers
-/sdk/communication/azure-communication-phonenumbers/                 @Arazan @whisper6284 @danielortega-msft @sofiar-msft
+/sdk/communication/azure-communication-phonenumbers/                 @gfeitosa-msft @phermanov-msft @ilyapaliakou-msft @besh2014 @ihuseynov-msft @kirill-linnik
```

Comment:
@kirill-linnik needs to be a public member of Azure to be added as code owner. See details here: https://github.com/Azure/azure-sdk-tools/blob/main/tools/codeowners-utils/README.md#user-is-not-a-public-member-of-azure